js/constants/console.js

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const strings = {
+  // Noscript
+  SCRIPT_BLOCKED: 'violates the following Content Security Policy directive: "script-src \'none\'"'
```

Comment:
it's a console string from blink (not us) and most users will never see it; not sure if localization is needed